docs/source/python_api/mlflow.metrics.rst

```diff
@@ -86,8 +86,38 @@ We provide the following builtin factory functions to create :py:class:`Evaluati
 
 .. autofunction:: mlflow.metrics.rougeLsum
 
+Retriever Specific Metrics
```

Comment:
we are moving the LLM autoeval metrics to a `mlflow.metrics.genai` namespace in https://github.com/mlflow/mlflow/pull/10138 to indicate that these are different. should we do the same for retriever metrics? else this `mlflow.metrics` module will get very large. i dont think we have to block on this btw.

Reply:
not blocking, but made a jira issue here for thinking abt [this here](https://databricks.atlassian.net/browse/ML-36139)